testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1563,3 +1563,16 @@
   raise 'File injection failed' unless return_code.zero?
   node.run('salt-call --local --file-root=/usr/share/susemanager/salt --module-dirs=/usr/share/susemanager/salt/ --log-level=info --retcode-passthrough state.apply ' + state)
 end
+
+When(/^I copy autoinstall mocked files on server$/) do
+  target_dirs = "/autoinstall/Fedora_12_i386/images/pxeboot/,/autoinstall/SLES15-SP2-x86_64/DVD1/boot/x86_64/loader/,/autoinstall/mock"
+  $server.run("mkdir -p {#{target_dirs}}")
+  current_dir = File.dirname(__FILE__)
```

Comment:
add to current_dir the constant part `upload_file/autoinstall/cobbler/` (name it `base_dir` or whatever)